src/Plutarch/Extra/FixedDecimal.hs

```diff
@@ -93,3 +141,47 @@ decimalToAdaValue =
         plam $ \(pto -> dec) ->
             let adaValue = (pdiv # dec # (pconstant (natVal (Proxy @unit)))) * pconstant 1000000
              in psingletonValue # pconstant "" # pconstant "" #$ adaValue
+
+{- | Convert @PInteger@ to @PFixedDecimal@.
+
+ @since 1.0.0
+-}
+fromPInteger ::
+    forall (unit :: Nat) (s :: S).
+    KnownNat unit =>
+    Term s (PInteger :--> PFixedDecimal unit)
+fromPInteger =
+    phoistAcyclic $ plam $ \z -> fromPInteger' z
+
+{- | Convert @PInteger@ to @PFixedDecimal@ in Haskell level.
+
+ @since 1.0.0
+-}
+fromPInteger' ::
+    forall (unit :: Nat) (s :: S).
+    KnownNat unit =>
+    Term s PInteger ->
+    Term s (PFixedDecimal unit)
+fromPInteger' z = pcon . PFixedDecimal $ pconstant (natVal (Proxy @unit)) * z
```

Comment:
Is there a reason we need both of these? I feel we can just have the Plutarch-level function only, and then use `#` in cases where we need a `Term`-based function.

Reply:
Looking back, we really don't them these.